src/test/java/CustomTests.java

```diff
@@ -12,52 +12,37 @@ public void ShuffleTest() {
 
         AllSongsPage allSongsPage = new AllSongsPage(driver);
         LoginPage loginPage = new LoginPage(driver);
-        BasePage basePage = new BasePage(driver);
+        HomePage homePage = new HomePage(driver);
 
         loginPage.login();
-        basePage.clickOnAllSongs();
+        homePage.clickOnAllSongs();
         allSongsPage.shuffleSongs();
     }
 
     @Test
     public void PlaySong() {
```

Comment:
**I recommend to use in the methods naming:**

- Naming functions: 
- Verb
- long names

Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized

For example 

```
   @Test
    public void isSongPlayingTest ()
```